kafka-ui-react-app/src/components/Connect/List/__tests__/ListItem.spec.tsx

```diff
@@ -52,6 +52,13 @@ describe('Connectors ListItem', () => {
     expect(screen.getAllByRole('cell')[6]).toHaveTextContent('2 of 2');
   });
 
+  it('topics tags are sorted', () => {
+    render(setupWrapper());
+    expect(screen.getAllByRole('link')[1]).toHaveTextContent('a');
```

Comment:
and here we can put it in a variable and reuse it for array , in order not to call it everytime

Reply:
fixed